py/selenium/webdriver/chrome/webdriver.py

```diff
@@ -115,6 +115,28 @@ def set_network_conditions(self, **network_conditions):
             'network_conditions': network_conditions
         })
 
+    def send_command_and_get_result(self, cmd, cmd_args):
```

Comment:
This is really a poor API name since most methods in any language are essentially send a command and get a result

Reply:
shouldn't we be consistent with [chromedriver interface name](https://cs.chromium.org/chromium/src/chrome/test/chromedriver/server/http_handler.cc?g=0&l=566)? 
or change to "send_devtools_command_and_get_result" is ok?